### DISCUSSION TITLE
Unable to install CLI.

### DISCUSSION THREAD

**Discussion by gbbest15** [saran-surya/email_auth_node#3] [Q&A]
i cant install the email-auth-node, is saying command not found

**Comment by saran-surya** [comment]
Hi, @gbbest15 so did you perform ```npm link``` ? After performing ```npm i``` ?

If you follow the steps on the setup page you should be good to go, Kindly check the step (2) and revert back again of you are facing issues.

https://saran-surya.github.io/email-auth-node/

**Comment by gbbest15** [comment]
![Uploading 16306092177154604532349498648590.jpg…]()

**Reply by saran-surya** [reply]
I am not able to open the image, can you clarify me that you are in the root of the project folder ?

And did you try running npx email-auth-node after performing npm link


And kindly copy paste the message that npm link gives you, (you can remove paths of any )

**Reply by gbbest15** [reply]
Okay I will try...

**Reply by saran-surya** [reply]
Yes please, and can we continue the discussion in one particular answer, as it is difficult to keep track of everything, I suggest we use this answer

**Reply by saran-surya** [reply]
Also, did you import this repo? Or clone it.

**Reply by saran-surya** [reply]
Hi @gbbest15 , Is there any update. Or can we close the discussion ?

**Reply by gbbest15** [reply]
i am still having the problem... i import this repo, after i clone it to my machine, the npm i && npm link.... is given me error

**Reply by gbbest15** [reply]
i will send my terminal picture as link so that it will be easy to view...

**Reply by gbbest15** [reply]
[https://drive.google.com/file/d/1C1_2ZQGIjsAqpx0hlhjgT4iskxNluepV/view?usp=sharing](url)
check this link to see the image of the terminal

**Reply by saran-surya** [reply]
Kindly check whether, the installed files are in the same user as the system user, if the files are installed to a different folder, then there will be issues. As I see, the npm link is working fine, and it is linking the email-auth-node to the npm/bin under a location of a user, Kindly check of the users are same.

**Reply by gbbest15** [reply]
okay.. i will check =..

**Reply by saran-surya** [reply]
Hi @gbbest15, Is your issue resolved ?

**Reply by gbbest15** [reply]
still having the issue.....

**Reply by saran-surya** [reply]
Did you check the user the package was installed ?,

**Reply by gbbest15** [reply]
i cant find it, can you connect to my system through zoom to solve this for me ?

**Reply by saran-surya** [reply]
You will have to check the npm folder under the user, in system settings, That nom folder will have a sub folder named npm. Then if you successfully navigate to the bin, you should be able to see the available global commands.
